builder/converter/sectioned_view.py

```diff
@@ -0,0 +1,184 @@
+import os
+from column_mapping import mapping
+import xml.etree.ElementTree as ET
+
+
+def convert_to_xml(yaml_dict):
+    root = get_xml_from_template('sectioned_view_template.xml')
+    set_name(root, yaml_dict)
+    set_sections(root, yaml_dict)
+    return root
+
+
+def get_xml_from_template(template_name):
+    current_dir = os.path.dirname(os.path.realpath(__file__))
+    template_path = os.path.join(current_dir, 'templates/%s' % template_name)
+
+    return ET.parse(template_path).getroot()
+
+
+def set_sections(root, yaml_dict):
+    sections = root.find('sections')
+    dict_sections = yaml_dict.get('sections')
+    if dict_sections:
```

Comment:
Do we want to throw an exception if dict_sections is not found in the yaml file? Or some kind of feedback to the user?